src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -558,7 +542,7 @@ tests:
           field set to G1 and the scene ID field set to 0x01."
       PICS: S.S.C05.Rsp
       command: "RecallScene"
-      groupId: 0x0001
+      groupId: 0x101
```

Comment:
Shouldn't this be using G1 instead of hard-coding the value?  Same for a bunch of other places here...

Reply:
As per above, I can add a check for S.S.C01.R, but executing the two tests seems redundant as they write the same scene to memory and will overwrite each other.

Reply:
It should, problem is I couldn't figure out how to get it to accept a variable.
Is this supported?

Reply:
Ah, this is groupId for the command, not a payload value?  In that case it might not work, in which case we should:

1. Hardcode for now.
2. File an issue on the runner to support it.

Reply:
Issue [here](https://github.com/project-chip/connectedhomeip/issues/29637), commend added.